wiki/Modding/NAT_meetings/id.md

```diff
@@ -0,0 +1,40 @@
+---
+tags:
+  - NAT/BN meetings
+  - Nomination Assessment Team meetings
+---
+
+# Pertemuan NAT
+
+**Pertemuan NAT** adalah diskusi terarah yang diselenggarakan oleh [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) (*NAT*) di channel `#modding` pada [server Discord osu!dev](/wiki/osu!dev_Discord_server). Topik pertemuan berkaitan dengan ekosistem [mapping](/wiki/Beatmapping) dan [modding](/wiki/Modding), termasuk masalah, solusi yang berpotensi, ide baru, dan laporan perkembangan. Meskipun kontributor terbanyak dalam diskusi ini adalah anggota NAT atau [Beatmap Nominator](/wiki/People/The_Team/Beatmap_Nominators), siapapun diperbolehkan untuk berpartisipasi.
```

Comment:
uhhhh i don't think `Meeting` here translates into `Pertemuan` tho, something like `Rapat Terbuka` or even `Musyawarah` is much more appropriate :(

``**Rapat Terbuka NAT** merupakan suatu diskusi terarah yang diselenggarakan oleh [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) (*NAT*) di channel `#modding` pada [server Discord osu!dev](/wiki/osu!dev_Discord_server). Setiap rapat yang diselenggarakan pada umumnya mengangkat topik bahasan yang berkaitan dengan ekosistem [mapping](/wiki/Beatmapping) dan [modding](/wiki/Modding) yang di dalamnya mencakup antara lain berbagai permasalahan terkini, solusi, ide-ide baru, dan laporan perkembangan. Meskipun rapat ini diutamakan bagi anggota NAT atau [Beatmap Nominator](/wiki/People/The_Team/Beatmap_Nominators), siapapun diperkenankan untuk ikut serta di dalamnya.``